ingestion/src/metadata/profiler/metrics/composed/null_ratio.py

```diff
@@ -52,6 +52,7 @@ def fn(self, res: Dict[str, Any]) -> Optional[float]:
 
         count = res.get(Count.name())
         null_count = res.get(NullCount.name())
-        if count + null_count == 0:
+        # Return None if count or null_count is None, or if their sum is zero
+        if count is None or null_count is None or count + null_count == 0:
```

Comment:
this should never be null. if it is then the method should fail and not hide it with `None`

Reply:
I will check where the issue is, it shouldn't return None, it should return 0..not sure if the interpretation within sqlalchemy is returning it 🤔